test/sqllogictest/ldbc_bi.slt

```diff
@@ -2866,52 +2866,49 @@ Explained Query:
             Map ((#4 OR #15)) // { arity: 17 }
               Join on=(#0 = #6 = #12 AND #1 = #7 = #13 AND #2 = #8 = #14 AND #3 = #9 AND #4 = #10 AND #5 = #11) type=delta // { arity: 16 }
                 implementation
-                  %0:l17 » %1:l10[#0..=#5]UKKKKKK » %2[#0..=#2]KKK
-                  %1:l10 » %0:l17[#0..=#5]KKKKKK » %2[#0..=#2]KKK
-                  %2 » %0:l17[#0..=#2]KKK » %1:l10[#0..=#5]UKKKKKK
+                  %0:l16 » %1:l16[#0..=#5]KKKKKK » %2[#0..=#2]KKK
+                  %1:l16 » %0:l16[#0..=#5]KKKKKK » %2[#0..=#2]KKK
+                  %2 » %0:l16[#0..=#2]KKK » %1:l16[#0..=#5]KKKKKK
```

Comment:
@ggevay is it right that the `U` going away reflects losing a unique key?

Reply:
Yes, U is an abbreviation for unique key, see [here](https://materialize.com/docs/sql/explain-plan/#reading-decorrelated-and-optimized-plans) in the EXPLAIN docs.

Reply:
Thanks! My guess is that this is a function of .. `join_implementation.rs` using `typ()` rather than the `UniqueKeys` analysis, as looking at the input (`l16`) it is headed by a `Distinct project=[#0..=#5]` which should make this be a `U`. This used to be `l17`, which also didn't have a `U` (it comes after this CTE term).

Reply:
With the `normalize_lets.rs` change, all `ldbc_bi` joins either preserve the `UK*` property or the join is substantially simplified (e.g. at least one term removed, usually down to 2-way join).